src/content/docs/faq.md

```diff
@@ -3,6 +3,8 @@ id: faq
 title: Frequently Asked Questions
 ---
 
+Here are some common questions that users have and their answers!
+
 ## How do I add plugin X to my configuration?
 
 AstroNvim provides a community-driven project called [AstroCommunity](https://github.com/AstroNvim/astrocommunity) to which every user is able to contribute plugin configurations. It is recommended to look there first since you can decide to either import a plugin specification if it's available or use it as a basis for your own customized specification.
```

Comment:
this should link to the documentation of astrocommunity

Reply:
Resolved in latest commit